compiler/optimizer/RematTools.cpp

```diff
@@ -290,7 +389,7 @@ void RematTools::walkTreeTopsCalculatingRematFailureAlternatives(TR::Compilation
          traceMsg(comp, "  priv arg remat: visiting [%p]: isStore %d privArg %d failedTargetMatch %d", start->getNode(),
             start->getNode()->getOpCode().isStoreDirect(),
             start->getNode()->getSymbol() && start->getNode()->getSymbol()->isAuto(),
-            failedTargets.ValueAt(start->getNode()->getFirstChild()->getGlobalIndex()));
+            start->getNode()->getNumChildren() ? failedTargets.ValueAt(start->getNode()->getFirstChild()->getGlobalIndex()) : -1);
```

Comment:
can we please make the > 0 explicit here since that is the intention - it isn't truly doing a truth test.